app/src/sharedTest/java/org/oppia/android/app/home/RecentlyPlayedFragmentTest.kt

```diff
@@ -463,7 +630,15 @@ class RecentlyPlayedFragmentTest {
   }
 
   @Test
-  fun testRecentlyPlayedTestActivity_configChange_lessonThumbnailIsCorrect() {
+  fun testRecentlyPlayedTestActivity_configChange_lessonThumbnailIsCorrect() { // ktlint-disable max-line-length
```

Comment:
/ ktlint-disable max-line-length not required i guess

Reply:
Removed.